lib/public/components/common/table/table.js

```diff
@@ -123,7 +124,7 @@ export const table = (
     }
 
     // Extract the profile of the table
-    const { profile: currentProfile = profiles.none } = tableConfiguration || {};
+    const { profile: currentProfile = profiles.none, xScrollable = false } = tableConfiguration || {};
```

Comment:
This is my personal feeling, I would rather name it "horizontalScroll"

Reply:
applied